frontend/ui/widget/opdsbrowser.lua

```diff
@@ -77,10 +77,14 @@ function OPDSBrowser:init()
           },
           {
              title = "Internet Archive",
-             url = "http://bookserver.archive.org/catalog/",
+             url = "https://bookserver.archive.org",
           },
         }
         G_reader_settings:saveSetting("opds_servers", servers)
+    else
```

Comment:
Slightly more elegant with just `elseif`. ;-)

But is this meant to be backward compatible with old configs? Should we spit out some debug message about that then?

Reply:
Of course :)